options/linux/include/sys/syscall.h

```diff
@@ -2,5 +2,9 @@
 #define _SYS_SYSCALL_H
 
 // OpenSSL wants this header to exist
+#ifdef __linux__
+#include <abi-bits/arch-syscall.h>
+#include <asm/unistd.h>
+#endif
```

Comment:
The comment is outdated, right? Also, `asm/unistd.h` is provided by the Linux kernel headers, right?

Reply:
`asm/unistd.h` provides syscall numbers in the form e.g `__NR_write` while `sys/syscall.h` provides syscall numbers in the form `SYS_write`. Different applications require different ones, but yes the kernel provides `asm/unistd.h`.